public/gbm_cptac_2021/meta_lipidome_negative_quantification.txt

```diff
@@ -0,0 +1,10 @@
+genetic_alteration_type: GENERIC_ASSAY
+show_profile_in_analysis_tab: true
+data_filename: data_lipidome_negative_quantification.txt
+datatype: LIMIT-VALUE
+stable_id: lipidome_negative
+profile_name: Lipid abundance ratio
+profile_description: Abundance log2 ratio for lipids measured by mass spectrometry (negative method).
```

Comment:
might be my ignorance, but what is negative method? Does it mean they only measured negatively-charged lipids?

Reply:
From the paper Liang-Bo Wang 2021: "Eluting lipids were introduced to the MS via electrospray ionization in both positive and negative modes [...]". I don't know a lot about MS, but it seems like there are two ways of ionizing the molecules before the analysis. And some lipids might be detected better in one mode or another. I found [this _very detailed_ article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4376555/).

I'll change method -> mode just in case 🤔